### ISSUE TITLE
Chrome 69 breaks the extension?

### ISSUE THREAD

**Issue by gregsadetsky** [gregsadetsky/chrome-dont-add-custom-search-engines#19]
I just received this over email:

Ever since the chrome 69 update, search engines are getting through the filter. Specifically Yahoo! & AOL. Still a great app and there are far fewer SE's to clean out now, however, I would love if you could find a way to block these too.

**Comment by cxw42** [comment]
I'm running the latest from @Procyon-b (#17) and it seems to block AOL and Yahoo OK in a quick test.  I still haven't had a chance to do a complete test, but do you think it's worth deploying anyway?

**Comment by gregsadetsky** [comment]
Yeah, I'd love to do this in the next few days. I couldn't find the time to do this earlier unfortunately.

**Comment by Procyon-b** [comment]
I have also tested against sites in this list, and it works ok.
Those using the extension have a very old version of the script. Without the aol and duckduckgo fixes.

**Comment by Procyon-b** [comment]
As a sidenote, has one of you been able to prevent chrome from updating?
I have talked about this with a friend this week-end, and he told me that he can (he is under win10).
Anyway, I was able to get rid of material design, so...

**Comment by cxw42** [comment]
I have not been able to prevent the update.  At this point I am resigned to breakage.  I try to use only extensions that are either actively supported by a large organization, or open-source so I can fix them :) .

**Comment by Procyon-b** [comment]
When I heard that chrome 69 was introducing an interface redesign to material design, I was a little bit worried. I have seen sites becoming almost unusable after (badly) moving to material design principle.
I don't have a link to it anymore, but I have seen one, with an almost empty page, except for a logo, a couple of link, and a big input field. The input field was hardly visible: white on a white page, without colored border, and a small very light grayish shadow under a small part of the bottom right corner, and no autofocus.
Then I saw the redesign of the web store...
But thanks to the flags page, I'm able to continue using chrome with pre-69 design.   :)

**Comment by gregsadetsky** [comment]
Oh boy. https://news.ycombinator.com/item?id=18064537

**Comment by cxw42** [comment]
@gregsadetsky We can close this, right?  I see you did a release on Sep. 25

@gregsadetsky @Procyon-b Any interest in joining me in a GitHub organization for Chrome extensions?  My TabFern extension is getting big enough to benefit from one, and I thought if you were interested I could create one and we could host both of our extensions there.  Any thoughts either way?
